nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const reservations = require("./../data/reservations.json");
+
+router.get("/", async (request, response) => {
+    response.json(reservations);
+  });
+  router.get("/:id", async (request, response) => {
+    const reservationWithid = reservations.find((reserve)=> `${reserve.id}` === request.params.id)
```

Comment:
This is nice and quite smart but I would rather the other way around, always try and alter the user's input to match your db/data than change your db/data to match the users input.

It's not a critique and I don't think you should change this, I just wanted to point it out.